source/Service.js

```diff
@@ -489,11 +522,10 @@ class Service extends EventEmitter {
             job.predicate.attemptsMax = job.attempts + 1;
         }
         job.status = JOB_STATUS_PENDING;
-        if (job.result.type !== JOB_RESULT_TYPE_FAILURE_SOFT) {
-            job.result.type = JOB_RESULT_TYPE_FAILURE_SOFT;
-        }
+        job.result.type = null;
         await this.storage.setItem(job.id, job);
         this.emit("jobReset", { id: job.id });
+        await this.logger.addEntry(this.logger.levels.LOGGER_INFO, `Reset job ${job.id}`);
```

Comment:
What if instead if adding these, we make the logger _listen_ for them instead? That way the service doesn't need to know about logging. It could be better so that we have the option to disable logs if we need to. The logger could listen to events like `jobReset`, a line above..

Reply:
Added, also made the logger listen to all events on init except taskScheduled which seems to appear a bit frequently to be useful, but it can be added there later if people want